sql/core/src/test/resources/sql-tests/inputs/decimals.sql

```diff
@@ -0,0 +1,16 @@
+-- tests for decimals handling in operations
+-- Spark draws its inspiration byt Hive implementation
```

Comment:
The hyperlinks in the PR cames from Microsoft, and the first purpose is SQL compliant. Can we remove this line?